src/syntax.js

```diff
@@ -51,6 +51,41 @@ function isFalsy (val) {
   return val === false || undefined === val || val === null
 }
 
+function validateExpression(exp, scope, errors = []) {
+  assert(scope, 'unable to evalExp: scope undefined')
+  var operatorREs = lexical.operators
+  var match;
+  for (var i = 0; i < operatorREs.length; i++) {
+    var operatorRE = operatorREs[i]
+    var expRE = new RegExp(`^(${lexical.quoteBalanced.source})(${operatorRE.source})(${lexical.quoteBalanced.source})$`)
+    if ((match = exp.match(expRE))) {
```

Comment:
can u autoformat this new code only?

like theres double `((` here for no reason